src/test/java/tests/DownloadCountingOutputStreamTest.java

```diff
@@ -0,0 +1,39 @@
+package tests;
+
+import javafx.beans.property.DoubleProperty;
+import javafx.beans.property.SimpleDoubleProperty;
+import org.junit.Test;
+import updater.DownloadCountingOutputStream;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+/**
+ * Test DownloadCountingOutputStream
+ */
+public class DownloadCountingOutputStreamTest {
+    @Test
+    public void downloadCountingOS_downloading_updateListener() {
```

Comment:
Yes, I was afraid of the confusion, but afraid of name being too long. Will spell out `OS`.